openssl/src/rsa.rs

```diff
@@ -465,6 +465,58 @@ impl Rsa<Public> {
     }
 }
 
+pub struct RsaKeyBuilder<Private> {
```

Comment:
This syntax makes a type parameter named `Private` which is probably not what's intended?

Reply:
I've renamed the field to `RsaPrivateKeyBuilder`